### DISCUSSION TITLE
Window overlaying FCPX

### DISCUSSION THREAD

**Discussion by trainspotta** [CommandPost/CommandPost#3272] [General]
Hello!

Is use commandpost with Loupedeck on FCPX.
As soon as I move the Playhead a huge black window open, just saying "PING". When I close it, it opens back up again.

How I can remove that please?
<img width="1409" alt="Bildschirmfoto 2023-09-29 um 13 21 08" src="https://github.com/CommandPost/CommandPost/assets/15868206/8c8e6a9b-9fa8-476b-b13d-cd8a0a9daf8b">

**Comment by latenitefilms** [comment]
Just make the Workflow Extension as small as it can go, and drag somewhere offscreen. See:

https://commandpost.io/final-cut-pro/workflow-extension/
